rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -50,6 +50,9 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
 RUN R -e "install.packages('Rcpp')"
+ARG NEEDPACKAGE
```

Comment:
Could you move the `NEEDPACKAGE` argument on the top of the dockerfile? maybe under line 2